copilot/explain.py

```diff
@@ -8,13 +8,13 @@
 class PiecesExplainCommand(sublime_plugin.TextCommand):
 	def run(self,edit):
 		copilot.clear()
-		if self.view.name():
-			ext = self.view.name().split(".")[-1]
+		if self.view.file_name():
+			ext = self.view.file_name().split(".")[-1]
```

Comment:
any reason why you have a -1 for the this?? would this be an index out of bounds?

Reply:
last index in the arry in python, learning something new for me